docs/rules/no-assignment-of-untracked-properties-used-in-tracking-contexts.md

```diff
@@ -69,6 +69,61 @@ class MyComponent extends Component {
 
 The autofixer for this rule will update assignments to use `set`. Alternatively, you can begin using tracked properties.
 
+## Configuration
+
+* object -- containing the following properties:
+  * array -- `extraMacros` -- array of configurations for custom macros, each with the following properties:
+    * string -- `name` -- The name the macro is exported with
+    * string -- `path` -- The file path used for importing the macro
+    * string -- `indexName` -- If this macro can also be imported through an index (like `computed` for `computed.and`), include it here
+    * string -- `indexPath` -- The path for importing the index
+    * array -- `argumentFormat` -- array of configurations for how to parse the arguments of the macro to extract the computed dependencies, with at least one of the following properties:
+      * object -- `strings` -- Configuration for extracting raw strings from the argument list, with the following options:
+        * number -- `count` -- How many arguments to consider as dependencies. Use `Number.MAX_VALUE` for all of them.
+        * number -- `startIndex` -- Defaults to zero. If it's something else, that many arguments will be skipped before checking for `count` dependencies.
+      * object -- `objects` -- Configuration for extracting the values of an object as dependency keys, with the following properties:
+        * number -- `index` -- The index of the argument to be checked.
+        * array -- `keys` -- Array of strings for which keys values should be checked for. If not provided, all values will be checked.
+
+Example configuration:
```

Comment:
Can you also show one of Ember's computed property macros as an example configuration? People are familiar with the ones from Ember so it would be easier for them to understand the configuration in terms of an example they're familiar with.